css-three-column-layout/styles.css

```diff
@@ -0,0 +1,65 @@
+* {
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  background-color: #1996da;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 750px;
+  margin: 100px auto;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
```

Comment:
You may not always want your rows to have their content set to space-between like this. I would move this to a separate class, probably named `justify-space-between`